src/main/java/org/cancogenvirusseq/muse/service/SubmissionService.java

```diff
@@ -130,7 +125,11 @@
                                     submission ->
                                         SubmissionEvent.builder()
                                             .submissionId(submission.getSubmissionId())
-                                            .userId(userId) // todo: auto UUID::fromString
+                                            .userId(
+                                                UUID.fromString(
+                                                    securityContext
+                                                        .getAuthentication()
+                                                        .getName())) // todo: auto UUID::fromString
```

Comment:
basically I was thinking of making a method like `securityContext -> UUID.from...(...getName())` as we do it twice in this class alone